pytorch_lightning/trainer/training_loop.py

```diff
@@ -574,6 +572,20 @@ def optimizer_closure():
 
         return 0, grad_norm_dic, all_log_metrics
 
+    def run_training_teardown(self):
```

Comment:
what about making it protected `_teardown_training`?

Reply:
what's the benefit? i think it's cleaner to be symmetric with the other methods (eg. `run_training_epoch`, `run_training_batch`, `run_training_teardown`)

Reply:
that make sense... maybe I would start a discussion if these kinds of workaround methods shall start with run_... but maybe its just my weakness in English :D

Reply:
haha, no worries 😄